keyboards/hazel/chaz/promicro/info.json

```diff
@@ -0,0 +1,64 @@
+{
+    "manufacturer": "Hazel",
+    "keyboard_name": "Chaz",
+    "maintainer": "jasonhazel",
+    "url": "https://github.com/hazels-garage/chaz",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "nkro": false
+    },
+    "matrix_pins": {
+        "cols": ["B5", "B4", "E6", "D7", "C6"],
+        "rows": ["D3", "D1", "D4", "F5", "D2", "D0", "F6", "F4"]
+    },
+    "development_board": "promicro",
+    "usb": {
+        "vid": "0x4A48",
+        "pid": "0x435A",
```

Comment:
`vid` and `pid` are identical for both `rp2040ce` and `promicro` directories, would you be able to move these two lines to `info.json` in the directory above (keyboard level)?